goci-interfaces/goci-ui/src/main/java/uk/ac/ebi/spot/goci/ui/component/ImpcIdentifierMap.java

```diff
@@ -0,0 +1,49 @@
+package uk.ac.ebi.spot.goci.ui.component;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class ImpcIdentifierMap {
+
+    private static final Logger log = LoggerFactory.getLogger(ImpcIdentifierMap.class);
+    private Map<String, String> impcGeneMap;
+
+    /** Populate a Singleton Hash Map with  gene as key &
+     * MGIdentifier as Value
+     *
+     * @param file
```

Comment:
I think the '@param file' & @return can be removed as the tag description is missing